cardano-explorer/cardano-explorer.cabal

```diff
@@ -72,6 +77,7 @@ executable cardano-explorer
   hs-source-dirs:      app
 
   ghc-options:          -Wall
+                        -Werror
```

Comment:
I certainly don't mind those there, but it can be done once eash in the `stack.yaml` and `cabal.project` files which is a little easier.